imgui.go

```diff
@@ -658,3 +658,36 @@ func IsItemHovered() bool {
 func IsKeyPressed(key int) bool {
 	return C.iggIsKeyPressed(C.int(key)) != 0
 }
+
+// ProgressBarV creates a progress bar.
+// size (for each axis) < 0.0f: align to end, 0.0f: auto, > 0.0f: specified size
+func ProgressBarV(fraction float32, size Vec2, overlay string) {
+	sizeArg, _ := size.wrapped()
+	overlayArg, overlayFin := wrapString(overlay)
+	defer overlayFin()
+	C.iggProgressBar(C.float(fraction), sizeArg, overlayArg)
+}
+
+// ProgressBar calls ProgressBarV(fraction, Vec2{-1, 0}, "")
+func ProgressBar(fraction float32) {
+	ProgressBarV(fraction, Vec2{-1, 0}, "")
+}
+
+// AlignTextToFramePadding vertically aligns upcoming text baseline to
+// FramePadding.y so that it will align properly to regularly framed
+// items (call if you have text on a line before a framed item)
+func AlignTextToFramePadding() {
+	C.iggAlignTextToFramePadding()
+}
+
+// SetIniFileName sets file name used to load and save
+// windows layout. Call with an empty string to disable INI file.
+// This function can be only called before first call to NewFrame()
```

Comment:
Although minimal, please add the documentation hint that calling this function with a string also creates a small memory leak.
The leak-free variant would be to use `LoadIniSettingsFromDisk()` and `SaveIniSettingsToDisk()`, though this then has to be done manually. Difficult decision :/

Reply:
The problem here is that for manual ini management IO.IniFilename must be set to NULL anyway, otherwise default ini file will be written periodically.

One possible solution is to use a static char array with size big enough to **hope** that user will never pass too long file name :)